### PR TITLE
feat(dunning): Add create payment requests endpoint

### PR DESCRIPTION
**Pull request by ancorcruz**
See https://github.com/getlago/lago-api/pull/2426

### CODE REVIEW

**Comment by nudded** [comment]
I forgot to write docs about this, but it's best you install `ruff` (https://docs.astral.sh/ruff/) locally. It will enforce consistent formatting 👍 I'll spend some time on writing some documentation about it.

**Review by nudded — APPROVED** [review]
LGTM!

File: lago_python_client/models/payment_request.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
 from typing import List, Optional
 
+from lago_python_client.base_model import BaseModel
```
Comment (nudded) [inline lago_python_client/models/payment_request.py:3]: this can be combined with the one below?

**Comment by ancorcruz** [comment]
> I forgot to write docs about this, but it's best you install `ruff` (https://docs.astral.sh/ruff/) locally. It will enforce consistent formatting 👍 I'll spend some time on writing some documentation about it.

Thanks, ruff is happy with the changes in the PR 🎉

**Comment by ancorcruz** [comment]
> Looks like it's not in agreement with the formatting :)

not in my machine 🤣 When I run `ruff check` finish with `All checks passed!`...

**Pull request merged**